geode-core/src/test/java/com/gemstone/gemfire/internal/offheap/OffHeapStoredObjectJUnitTest.java

```diff
@@ -816,6 +820,7 @@ public void getRawBytesShouldThrowExceptionIfValueIsCompressed() {
     chunk.getRawBytes();
 
     chunk.release();
+    fail("Expected getRawBytes() for a compressed value to throw java.lang.AssertionError");
```

Comment:
Why is this "fail" call added. Doesn't the "expected" on the Test annotation take care of this?